File: Week_01/1281.cs
public class Solution {
    public int SubtractProductAndSum(int n) {
        int sum = 0;
        int accumulate = 1;
        while(n!=0) {
            sum += n%10;
            accumulate *= n%10;
            n=(n-n%10)/10;
        }
        return accumulate - sum;
    }
}

File: Week_01/1281.go
func subtractProductAndSum(n int) int {
    var (
        sum = 0
        acm = 1
    )
    for n!=0 {
        sum+=n%10
        acm*=n%10
        n=(n-n%10)/10
    }
    return acm - sum
}